api/cron-jobs/projects.js

```diff
@@ -1,33 +1,34 @@
-require('dotenv').config();
-const airTable = require('../helpers/airTable');
-const arraysHelpers = require('../helpers/arrays');
-const axios = require('axios').default;
-const projectsHelpers = require('../helpers/projects');
-const videosService = require('../services/videos');
+require("dotenv").config();
```

Comment:
I think your code editor has automatically reformatted lots of single quotes to double quotes so it's showing up with more changes than necessary -- these should all stay as single quotes as that's the code style here.  (We have [a ticket in the backlog to set proper coding style config for the API code](https://sta2020.atlassian.net/browse/SVA-293), which would help avoid this happening but we haven't done that yet.)